### ISSUE TITLE
ActivityNotFoundException: android.intent.action.OPEN_DOCUMENT

### ISSUE THREAD

**Issue by AbdulrahmanKassem** [Dhaval2404/ImagePicker#6]
**ActivityNotFoundException: android.intent.action.OPEN_DOCUMENT**
```
Fatal Exception: java.lang.RuntimeException
:Unable to start activity ComponentInfo{com.github.dhaval2404.imagepicker.ImagePickerActivity}
:android.content.ActivityNotFoundException
:No Activity found to handle Intent { act=android.intent.action.OPEN_DOCUMENT cat=[android.intent.category.OPENABLE] typ=image/* }
```
I face this issue only in android 8 (oreo)

**Comment by Dhaval2404** [comment]
Thank you @AbdulrahmanKassem for trying out the lib.  I will fix this issue ASAP. 

Can you please let me know the Device you are testing.

**Comment by AbdulrahmanKassem** [comment]
Brand: LAVA
Model: Z61-2GB
Version: 8.1.0

Brand: LAVA
Model: Z61
Version: 8.1.0

**Comment by Dhaval2404** [comment]
Hello @AbdulrahmanKassem, Sorry for the late response. I haven't able to reproduce this issue but I have applied fixes to the hotfix/v1.1 branch. Can you please check the below APK to verify that the issue is resolved or not. 

[https://www.dropbox.com/s/q89fizga4ssz1ml/ImagePicker-v1.1.1.apk?dl=1](https://www.dropbox.com/s/q89fizga4ssz1ml/ImagePicker-v1.1.1.apk?dl=1)


Thank you.

**Comment by Dhaval2404** [comment]
Fixed in v1.2
